templates/faq/index.html.twig

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+	{{ parent() }}
+{% endblock %}
+{% block stylesheet %}
+	<link rel="stylesheet" href="assets/css/mocktail_details.css"/>
+{% endblock %}
+{% block header %}
+	<div class="page-heading header-text">
+		<div class="container">
+			<div class="row">
+				<div class="col-lg-12">
+					<h1></h1>
+				</div>
+			</div>
+		</div>
+	</div>
+{% endblock %}
+{% block content %}
+	<div class="single-mocktail section">
```

Comment:
no h1 in this page. + use html semantic for better referencing like section / ul & li etc...